src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.hpp

```diff
@@ -125,7 +125,11 @@ class ShenandoahGenerationSizer {
   // given the number of heap regions depending on the kind of sizing algorithm.
   void recalculate_min_max_young_length(size_t heap_region_count);
 
-public:
+  bool transfer_regions(ShenandoahGeneration* src, ShenandoahGeneration* dst, size_t regions) const;
```

Comment:
Transfer regions has a bit of nuance, in that it fails without affecting the transfer if the transfer won't succeed without violating source or destination size bounds. I'd explain when it returns true and when false in a documentation line here.